### PR TITLE
docs(template): add publish ticket type

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

**Fixes:** #134

## Summary:

Adds a new ticket template for tracking compliance necessary to publish an AuroLabs component to Auro Design System

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (new GitHub ticket template)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: template/.github/ISSUE_TEMPLATE/publish_component.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+name: Publish Component
+about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
+title: 'auro-[name]; publish to Auro Design System'
+labels: 'Status: Proposal, not-reviewed'
```
Comment (blackfalcon) [inline template/.github/ISSUE_TEMPLATE/publish_component.md:5]: Is the idea that either you or I would use this? If that is the case, then the default label should be something different I assume.

File: template/.github/ISSUE_TEMPLATE/publish_component.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- [ ] README doc completed with all relative information
+- [ ] API documentation created
+
+### Testing
+- [Browser compatibility](http://auro.alaskaair.com/support/browsersSupport) - desktop and mobile versions
+  - [ ] Chrome
+  - [ ] Firefox
+  - [ ] Edge (79 +)
+  - [ ] Safari
+- [Accessibility](http://auro.alaskaair.com/a11y-statement)
```
Comment (blackfalcon) [inline template/.github/ISSUE_TEMPLATE/publish_component.md:40]: Should be updated to say "one or more of the following"

File: template/.github/ISSUE_TEMPLATE/publish_component.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - [ ] Axe / Google Lighthouse (baseline standard)
+  - [ ] Screenreader (determine usability of experience via screenreader)
+  - [ ] Keyboard use (a user must be able to navigate to key parts via a keyboard)
+  - [ ] Accessibility should be verified against all browsers
+- [ ] Bundled versions tested
+
+### Review
+- [ ] Review demo with the designer as well as the Auro team
+- [ ] Review demo with Auro Team
+- [ ] All work to be reviewed by Auro team
```
Comment (blackfalcon) [inline template/.github/ISSUE_TEMPLATE/publish_component.md:50]:
```suggestion
- [ ] All work is reviewed by Auro team
```

File: template/.github/ISSUE_TEMPLATE/publish_component.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- [Accessibility](http://auro.alaskaair.com/a11y-statement)
+  - [ ] Axe / Google Lighthouse (baseline standard)
+  - [ ] Screenreader (determine usability of experience via screenreader)
+  - [ ] Keyboard use (a user must be able to navigate to key parts via a keyboard)
+  - [ ] Accessibility should be verified against all browsers
+- [ ] Bundled versions tested
+
+### Review
+- [ ] Review demo with the designer as well as the Auro team
+- [ ] Review demo with Auro Team
```
Comment (blackfalcon) [inline template/.github/ISSUE_TEMPLATE/publish_component.md:49]: Don't think we need this one?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 1 above)
Comment (jason-capsule42) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:5]: I assume that the component developer would create this ticket and largely use it as a checklist themselves. Certain list items obviously requiring them to coordinate with either of us / a designer in order to complete.

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 2 above)
Comment (jason-capsule42) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:40]: The browser list? Considering differences in Safari vs Everyone else and mobile vs. non-mobile experience do you think that testing against a single browser is suffecient?

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 4 above)
Comment (jason-capsule42) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:49]: My thinking is that everything before the "review" section is largely done without Auro Team involvement (probably some of course). An experienced Auro contributor shouldn't need much from us up until it's time to review.

The "review" section is really where we as the Auro Design System team are giving our final once over. While we may not need to check these off ourselves, I included them more as a complete list of everything the contributor will need to understand as "gate-keeper" tasks before becoming an official Auro component.

Thoughts?

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 1 above)
Comment (blackfalcon) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:5]:
```suggestion
labels: 'Status: Review Needed, not-reviewed'
```

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 2 above)
Comment (blackfalcon) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:40]: No, I was referring to the accessibility options.

File: template/.github/ISSUE_TEMPLATE/publish_component.md (hunk 4 above)
Comment (blackfalcon) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:49]: I can get on board with that, but we then need to clearly communicate that. Like some kind of instruction. From me reading this list, I didn't get that context.

Comment (jason-capsule42) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:49]: I've updated the language in the template to reflect this.

Comment (blackfalcon) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:49]: I am not seeing updated content?

Comment (jason-capsule42) [reply template/.github/ISSUE_TEMPLATE/publish_component.md:54]: Should be there now.

**Review by blackfalcon — DISMISSED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**